source/_integrations/zwave_js.markdown

```diff
@@ -959,10 +962,13 @@ Many users have reported issues with interference when the USB stick was directl
 
 ##### Enable Z-Wave JS logging
 
-1. Go to the Z-Wave integration panel: {% my integration badge domain="zwave_js" %}
-2. Select `Enable debug logging` on the left-hand side of the screen.
-
-The log level will be set to `debug` for the integration, library, and optionally the driver (if the driver log level is not already set to `verbose`, `debug`, or `silly`), and all Z-Wave JS logs will be added to the Home Assistant logs.
+1. Define the log level:
+   - By default, it is set to `info` for the integration, library, and optionally the driver (if the driver log level is not already set to `verbose`, `debug`, or `silly`), and all Z-Wave JS logs will be added to the Home Assistant logs.
```

Comment:
This sentence doesn't make sense now. Before your change we described what happened when the log was set to debug. We should retain that.